working-groups/community_meeting/26-01-2023.md

```diff
@@ -0,0 +1,47 @@
+# 26-01-2023 [community]  
+## Agenda - Open to Discussion
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
+- Example - Harsh is taking the blog on “DevOps practises” and “FLINT.Cloud GCP” by Gopi 
+- You being recognized by the name and able to learn lot’s of thing regarding the MojaGlobal just by writing about the same.
```

Comment:
Can you please correct from `lot's of thing` => `lot's of things`? and on the same line from `MojaGlobal` to `moja global`?